docs/env_vars.md

```diff
@@ -6,8 +6,6 @@ wlroots reads these environment variables
   libinput, drm, wayland, x11, headless, noop)
 * *WLR_NO_HARDWARE_CURSORS*: set to 1 to use software cursors instead of
   hardware cursors
-* *WLR_SESSION*: specifies the `wlr_session` to be used (available sessions:
```

Comment:
Nit: this belongs to the "Remove session_impl" commit.

Reply:
Split it for correctness so the "direct" commit removes the direct option, while the "session_impl" commit removes the env var.